daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/ValueEnricherSpec.scala

```diff
@@ -16,13 +15,13 @@ import org.scalatest.prop.TableDrivenPropertyChecks
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 
-import scala.language.implicitConversions
-
 class ValueEnricherSpec extends AnyWordSpec with Matchers with TableDrivenPropertyChecks {
 
-  import defaultParserParameters.{defaultPackageId => pkgId}
+  import TransactionBuilder.Implicits.{defaultPackageId => _, _}
+  import com.daml.lf.testing.parser.Implicits._
 
-  private implicit def toName(s: String): Ref.Name = Ref.Name.assertFromString(s)
+  private[this] implicit val defaultPackageId: Ref.PackageId =
```

Comment:
why do we need to change it instead of using the one from the builder?

Reply:
Both the builder and the parser need a default Package ID.
We need the both package IDs to have the same value, either we override the one form the builder  either we override the one of the parser.  I have done the former. The latter could be done with something like that:
```suggestion
  private[this] implicit val defaultParserParameters: parser.ParserParameters[this.type] =
    defaultParserParameters.copy(defaultPackageId = defaultPackageId)
```

Reply:
makes sense, so the actual package id doesn’t matter it just matters that they’re the same :+1:

Reply:
:+1: